configure.d/1_reread_partitions.conf

```diff
@@ -11,7 +11,12 @@ check() {
     config_file_path=$1
     if compile_module $cur_name "bdev_disk_changed(NULL, false);" "linux/genhd.h"
     then
-        echo $cur_name "1" >> $config_file_path
+        if compile_module $cur_name "bdev_disk_changed((struct block_device *)NULL, false);" "linux/genhd.h"
+        then
+            echo $cur_name "3" >> $config_file_path
+        else
+            echo $cur_name "1" >> $config_file_path
+        fi
```

Comment:
Is the indentation really necessary? Couldn't it be simply 
```
if ...
else if ...
else 
```

Reply:
fixed!